src/System.Threading/tests/Performance/Perf.Interlocked.cs

```diff
@@ -0,0 +1,127 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Xunit.Performance;
+
+namespace System.Threading.Tests
+{
+    public class Perf_Interlocked
+    {
+        [Benchmark]
+        public static void IncrementDecrement_int()
+        {
+            int integer32 = 0;
+            foreach (var iteration in Benchmark.Iterations)
+            {
+                using (iteration.StartMeasurement())
+                {
+                    Interlocked.Increment(ref integer32);
+                    Interlocked.Decrement(ref integer32);
```

Comment:
Would be good to do a fixed number of same operations per iteration such that the result is comparable across benchmarks, overflow shouldn't be an issue